src/ssl.c

```diff
@@ -6659,6 +6797,65 @@ int ProcessFile(WOLFSSL_CTX* ctx, const char* fname, int format, int type,
                                 userChain, verify);
     }
 
+#ifndef NO_WOLFSSL_CM_VERIFY
+    /* Call to over-ride status */
+    if ((ctx != NULL) && (ctx->cm != NULL) &&
+        (ctx->cm->verifyCallback != NULL)) {
+        int tmp_ret = 0;
```

Comment:
There are keys, encrypted keys, crls, and user chain certs at the least here that I don't see you removing.  That is, trying to parse a private key and calling cert verify override on it should be breaking lots of cases.  Something is wrong here.